ArduCopter/motors.cpp

```diff
@@ -303,7 +303,10 @@ void Copter::motors_output()
     // OBC rules
     if (g2.afs.should_crash_vehicle()) {
         g2.afs.terminate_vehicle();
-        return;
+        if (! g2.afs.terminating_vehicle_via_landing()) {
```

Comment:
there's a tiny little extra space after the "1".  I'm happy to merge anyway if you don't feel like fixing it..